uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -0,0 +1,251 @@
+<div class="modal-header">
```

Comment:
The original file,
https://github.com/civicrm/civihr/blob/PCHR-1302-AngularJS-LeaveRequestModal-directive/uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

Should be removed